packages/sanity/src/structure/panes/document/document-layout/DocumentLayout.tsx

```diff
@@ -249,6 +253,17 @@ export function DocumentLayout() {
             __unstable_elements={{[DOCUMENT_PANEL_PORTAL_ELEMENT]: documentPanelPortalElement}}
           >
             <DialogProvider position={DIALOG_PROVIDER_POSITION} zOffset={zOffsets.portal}>
+              {StartInCreateBanner && (
+                <ShowWhenPaneOpen>
+                  <StartInCreateBanner
+                    document={value}
+                    documentId={documentId}
+                    documentType={schemaType}
+                    isInitialValueLoading={!!isInitialValueLoading}
+                    panelPortalElementId={DOCUMENT_PANEL_PORTAL_ELEMENT}
+                  />
+                </ShowWhenPaneOpen>
+              )}
```

Comment:
Added a comment about the position of this [here](https://github.com/sanity-io/sanity/pull/7955#issuecomment-2523099739)